backend/src/main/java/codesquad/airbnb/accomodation/service/AccommodationService.java

```diff
@@ -0,0 +1,32 @@
+package codesquad.airbnb.accomodation.service;
+
+import codesquad.airbnb.accomodation.domain.Accommodation;
+import codesquad.airbnb.accomodation.dto.AccommodationResponse;
+import codesquad.airbnb.accomodation.dto.AccommodationSearchRequest;
+import codesquad.airbnb.accomodation.repository.AccommodationReservationQueryRepository;
+import codesquad.airbnb.reservation.Schedule;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class AccommodationService {
+
+    private final AccommodationReservationQueryRepository accommodationQueryRepository;
+
+    @Transactional
+    public List<AccommodationResponse> findNotReservedAccommodations(AccommodationSearchRequest searchRequest) {
+        Schedule schedule =
+                Schedule.of(searchRequest.getStartDate(), searchRequest.getEndDate(), searchRequest.getVisitors());
```

Comment:
파라미터 수를 줄일 수 있을것 같네요.
```suggestion
        Schedule schedule =
                Schedule.from(searchRequest);
```

Reply:
도메인 객체에게 dto를 숨기는것도 좋을듯합니다.
```suggestion
        Schedule schedule = searchRequest.toSchedule(); //내부에서 생성자 호출
```